<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Anatomist Simple Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../anatomist-free/branches/stable/shared/icons/anatomist.ico</normaloff>../../../../../../anatomist-free/branches/stable/shared/icons/anatomist.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Objects:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="objectslist">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Information:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="infoLabel">
            <property name="text">
             <string>none</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="windows">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>4</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="fileOpenAction"/>
    <addaction name="fileSaveAction"/>
    <addaction name="fileExitAction"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="editAddAction"/>
    <addaction name="editRemoveAction"/>
    <addaction name="editDeleteAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="helpIndexAction"/>
    <addaction name="separator"/>
    <addaction name="helpAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="viewEnable_Volume_RenderingAction"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="fileOpenAction"/>
   <addaction name="fileSaveAction"/>
   <addaction name="separator"/>
   <addaction name="editAddAction"/>
   <addaction name="editRemoveAction"/>
   <addaction name="editDeleteAction"/>
   <addaction name="separator"/>
   <addaction name="helpIndexAction"/>
  </widget>
  <action name="fileOpenAction">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../anatomist-free/branches/stable/shared/icons/anatomist.ico</normaloff>../../../../../../anatomist-free/branches/stable/shared/icons/anatomist.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="fileSaveAction">
   <property name="text">
    <string>&amp;Save...</string>
   </property>
  </action>
  <action name="fileExitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="editAddAction">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../../../../../../../home/appli/neurosvn/anatomist/anatomist-free/branches/stable/shared/icons/add.xpm</normaloff>../../../../../../../../../../../../home/appli/neurosvn/anatomist/anatomist-free/branches/stable/shared/icons/add.xpm</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="editRemoveAction">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../../../../../../../home/appli/neurosvn/anatomist/anatomist-free/branches/stable/shared/icons/remove.xpm</normaloff>../../../../../../../../../../../../home/appli/neurosvn/anatomist/anatomist-free/branches/stable/shared/icons/remove.xpm</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="editDeleteAction">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../../../../../../../tmp/delete.png</normaloff>../../../../../../../../../../../../tmp/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="helpIndexAction">
   <property name="text">
    <string>Index</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="helpAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="viewEnable_Volume_RenderingAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Volume Rendering</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
